nemo-physical/src/management/database/storage.rs

```diff
@@ -114,13 +112,13 @@ impl TableStorage {
 }
 
 impl ByteSized for TableStorage {
-    fn size_bytes(&self) -> ByteSize {
+    fn size_bytes(&self) -> u64 {
         match self {
             TableStorage::InMemory(trie) => trie.size_bytes(),
-            TableStorage::FromSources(sources) => {
-                sum_bytes(sources.iter().map(|source| source.size_bytes()))
-            }
-            TableStorage::Empty => ByteSize::b(0),
+            TableStorage::FromSources(sources) => sources
+                .iter()
+                .fold(0, |acc, source| acc + source.size_bytes()),
```

Comment:
why not something like this, which I think is more readable?

```rust
TableStorage::FromSources(sources) => sources.iter().map(
  |source| source.size_bytes()
).sum()
```

Reply:
If you think it is worth your time, feel free to replace all uses of fold for summation accordingly. I won't make changes in this branch now.